admin-site/src/ButtonActions.svelte

```diff
@@ -0,0 +1,9 @@
+<script context="module">
```

Comment:
So this executes in a global context?

Reply:
It allows for that script to get called/imported to other components. Not implemented, in the current state, just trying things out. Likely won't have to do much of it (if any) with functions in their respective components.